src/module-elasticsuite-indices/Controller/Adminhtml/Index/RemoveGhostIndices.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/**
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Smile ElasticSuite to newer versions in the future.
+ *
+ * @category  Smile
+ * @package   Smile\ElasticsuiteIndices
+ * @author    Vadym Honcharuk <vahonc@smile.fr>
+ * @copyright 2025 Smile
+ * @license   Open Software License ("OSL") v. 3.0
+ */
+
+namespace Smile\ElasticsuiteIndices\Controller\Adminhtml\Index;
+
+use Exception;
+use Magento\Backend\App\Action\Context;
+use Smile\ElasticsuiteIndices\Block\Widget\Grid\Column\Renderer\IndexStatus;
+use Smile\ElasticsuiteIndices\Controller\Adminhtml\AbstractAction;
+use Smile\ElasticsuiteIndices\Model\IndexStatsProvider;
+use Smile\ElasticsuiteIndices\Model\IndexStatusProvider;
+
+/**
+ * Controller for removing all ghost indices.
+ *
+ * @category Smile
+ * @package  Smile\ElasticsuiteIndices
+ * @author   Vadym Honcharuk <vahonc@smile.fr>
+ */
+class RemoveGhostIndices extends AbstractAction
+{
+    /**
+     * Authorization level of a basic admin session.
+     */
+    public const ADMIN_RESOURCE = 'Smile_ElasticsuiteIndices::remove_ghost_indices';
```

Comment:
Please use the existing ACL Smile_ElasticsuiteIndices::remove instead since it is already used to check if one can delete one (ghost only) index at a time.

Reply:
Done